### DISCUSSION TITLE
XNEdit for ARM64

### DISCUSSION THREAD

**Discussion by efa** [unixwork/xnedit#77] [Show and tell]
just for curiosity, I tryed to compile XNEdit on a Debian 10 installed on an ARM64 platform
(just in case is UserLAnd on Android on my Mobile).
XNEdit work well exporting the DISPLAY via SSH, see:
![xnedit_ARM64](https://user-images.githubusercontent.com/1578980/149037357-7e8ac88b-f476-42c9-ba2d-4af52371b483.png)
and reasonably well using a local X server on Android.

**Comment by unixwork** [comment]
Interesting, I did not know UserLAnd.

XNEdit also works fine on a raspberry pi.

**Comment by efa** [comment]
maybe you should mention the ARM target as supported architecture.
About macOS (amd64 and arm64), how do you build and test that target?

As soon as I have a RISC-V platform [1] in my hands, I will try to compile it there too
[1] https://en.wikipedia.org/wiki/RISC-V

**Comment by unixwork** [comment]
There is no CPU specific code in xnedit, so in theory it should work on all CPU platforms.

Is there any interesting RISC-V board available or in development?

**Comment by efa** [comment]
I will go for SiFive products:
https://www.sifive.com/boards
where HiFive1 Rev B is cheap but is 32bit of the ARM Cortex-M class, so seems not supported by Linux

Debian suggest some other:
https://wiki.debian.org/RISC-V#Hardware

**Comment by efa** [comment]
at the end, for global semiconductor shortage, I had to buy a board without wired ethernet and DDR 512 MB only, but 64 bit, 1 GHz and quite complete with WiFi, BT, HDMI 4k and HEVC real time decoding:
https://www.aliexpress.com/item/1005003741287162.html?spm=a2g0o.store_pc_groupList.8148356.13.3d616c1e34fJT6
at 33 euro delivery included.

**Comment by efa** [comment]
if you are interested in RISC-V, last 9 days:
https://www.kickstarter.com/projects/starfive/visionfive-2-lite-unlock-risc-v-sbc-at-199

I got the VisionFive 2 "non LITE" without WiFi one year ago, and it was a good product

Debian 13 natively support the SoC
